CHANGELOG.md

```diff
@@ -5,6 +5,101 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 8.9.0
+
+_Jul 17, 2025_
+
+We'd like to extend a big thank you to the 10 contributors who made this release possible. Here are some highlights ✨:
+
+- ✨ Improve the drag and drop interaction for Data Grid [row reordering](https://mui.com/x/react-data-grid/row-ordering/) feature. It uses a drop indicator to point to the position the row would be moving to.
+
+  https://github.com/user-attachments/assets/37284c4f-e8d4-4fc6-a6af-a780592905ef
+
+- 🚀 Improve Data Grid Pivoting and Aggregation performance
+
+- 📊 Add `minBarSize` to set a minimum height for bars
+
+Special thanks go out to the community members for their valuable contributions:
+@lauri865
+
+The following are all team members who have contributed to this release:
+@alexfauquette, @arminmeh, @bernardobelchior, @flaviendelangle, @JCQuintas, @LukasTy, @mapache-salvaje, @noraleonte, @MBilalShafi
+
+### Data Grid
+
+#### `@mui/x-data-grid@8.9.0`
```

Comment:
This version is shown here but it's not actually [published](https://www.npmjs.com/package/@mui/x-data-grid), causing https://github.com/mui/mui-x/issues/18847.

@bernardobelchior Could you confirm if you used the default suggestions by `lerna` or had to do some manual changes?
If it wasn't suggested in the first place I think it's something we need to investigate, as all the Data Grid packages should be published if there's a change in any of them.

Reply:
Option 1 sounds good if doable.

Reply:
Same opinion! If anyone can do that it would be great!

Reply:
Unfortunately, this is not an option anymore. 😞 
<img width="879" height="163" alt="image" src="https://github.com/user-attachments/assets/d26eb253-fbc3-4a9a-b1bc-f2939cc94515" />

We'll have to go with option 2.

Reply:
Hey! Sorry for the mess. 

I had realized the script didn't update `@mui/x-data-grid`'s version (see the first commit [here](https://github.com/mui/mui-x/pull/18827/commits/d5d26d832ed800444f4349d0e2e5d37b35b80a99#diff-972244d03053b5f6f0de27f05b46ab082c01c6d424576dcb35fe6f3b09b16297)), but only now realized that I forgot to update it manually. I must have forgotten to do it. 

I saw that you already released a new version. Thank you.

Reply:
> I had realized the script didn't update @mui/x-data-grid's version (see the first commit [here](https://github.com/mui/mui-x/pull/18827/commits/d5d26d832ed800444f4349d0e2e5d37b35b80a99#diff-972244d03053b5f6f0de27f05b46ab082c01c6d424576dcb35fe6f3b09b16297)), but only now realized that I forgot to update it manually

Do you have any idea why the script didn't update the data grid's version?
I usually trust the script and don't double-check 😅

Reply:
No 😅 

I no longer have the log, but [here's](https://github.com/mui/mui-x/pull/18827#discussion_r2210559851) the selection that I made at the time. 

Now that I look at it, the data grid package isn't there. I'm pretty sure I selected its correct version, but maybe I didn't? Not sure 😅

Reply:
That's odd, I just ran `pnpm release:version` and the only way I could keep the current version would be to select "Custom version" and type in the current version 🙃
<img width="446" height="135" src="https://github.com/user-attachments/assets/0df69477-e34e-4a29-b45e-0d84ec7253ac" />

Reply:
Yeah, there must have been some bug because I'm pretty sure I didn't do that